ext/mysqlnd/mysql_float_to_double.h

```diff
@@ -31,7 +31,7 @@
 
 /*
  * Convert from a 4-byte float to a 8-byte decimal by first converting
- * the float to a string, and then the string to a double.
+ * the float to a string (Iinoring localization), and then the string to a double.
```

Comment:
```suggestion
 * the float to a string (Ignoring localization), and then the string to a double.
```

Reply:
Good catch, thanks! Actually I wanted to lower-case the word after copy&paste, but that went wrong. Anyhow, fixed now.